packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.constants.js

```diff
@@ -0,0 +1,11 @@
+export const IG_MAP_URL = {
+  FR: 'https://www.ovhtelecom.fr/xdsl/incident/#/',
+};
+
+export const TRAVAUX_URL =
```

Comment:
may be we could use English keyword:
`JOBS_URL` or `TASKS_URL` instead of `TRAVAUX_URL`